Jerrod/Assignments/TDD/bank_account_tdd/ba_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'bankAccount'
+
+RSpec.describe BankAccount do
+  before do
+    @b = BankAccount.new
+    @a = BankAccount.new
+    @a.deposit('checking', 1000)
+    @a.deposit('savings', 1000)
+  end
```

Comment:
Why do you have an @a and an @b. You could have a before block for just a few tests if you wrap it in describe like below:

```ruby
describe 'behavior' do
  before do
    @a = something
  end

  it 'some test' do
  end
end

describe 'behavior2' do
  before do
    @a = something
  end

  it 'some test' do
  end
end
```